dynamic/utility.py

```diff
@@ -333,7 +333,7 @@ def __get_search_url(self, question, tags):
         data provided by the user such as tags and question, which
         can finally be used to get answers
         """
-        return f"{self.search_content_url}/2.2/search/advanced?order=desc&sort=relevance&tagged={tags}&title={question}&site=stackoverflow"
+        return f"{self.search_content_url}/2.2/search/advanced?order=desc&sort=relevance&tagged={tags}&q={question}&site=stackoverflow"
```

Comment:
Can you break this line into 2 ? PEP8 is complaining that this line is too long

Reply:
Did the required changes... Please check...